src/ui/StartUp.java

```diff
@@ -115,11 +115,12 @@ public static void main(String[] args) {
 	// starts the program
 	private void startProgram() {
 		// set indexes in the array of images
-		this.imageManager.setIndexImages(this.images);
+		this.imageManager.setIndexImages();
 		// shuffle the index position of each image
-		this.imageManager.shuffleIndexImages(this.images, numberOfImages);
+		this.imageManager.shuffleIndexImages(numberOfImages);
 		// get the location of the positive image
-		positiveImagePosition = this.imageManager.getPositiveImagePosition(this.images);
+		positiveImagePosition = this.imageManager.getPositiveImagePosition();
+		images = this.imageManager.returnArray();
```

Comment:
In urmatorul review, o sa facem cateva modificari pentru a nu mai fi nevoie de aceasta declaratie.

Reply:
la care declaratie te referi, la returnArray?

Reply:
Da.